src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -347,7 +347,17 @@ public String getRepositoryUri(@NonNull BitbucketRepositoryType type,
                 // This is required for Bitbucket Server to update the refs/pull-requests/* references
                 // See https://community.atlassian.com/t5/Bitbucket-questions/Change-pull-request-refs-after-Commit-instead-of-after-Approval/qaq-p/194702#M6829
                 if (endpoint.isCallCanMerge()) {
-                    pullRequest.setCanMerge(getPullRequestCanMergeById(pullRequest.getId()));
+                    try {
+                        pullRequest.setCanMerge(getPullRequestCanMergeById(pullRequest.getId()));
+                    } catch (BitbucketRequestException e) {
+                        // see JENKINS-65718 https://docs.atlassian.com/bitbucket-server/rest/5.16.0/bitbucket-rest.html#errors-and-validation
```

Comment:
Bitbucket 5.x has been EOL'd.  Does this also occur on 6.x or later?

Reply:
link with 7.2.1 says the same :) https://docs.atlassian.com/bitbucket-server/rest/7.2.1/bitbucket-rest.html#errors-and-validation

Reply:
link updated